docs/source/intro-clx-streamz.ipynb

```diff
@@ -39,140 +39,167 @@
-    "- `input_topic` - The name for the input topic to send the input dataset. Any name can be indicated here.\n",
-    "- `output_topic` - The name for the output topic to send the output data. Any name can be indicated here.\n",
-    "- `model_file` - The path to your model file\n",
-    "- `label_file` - The path to your label file\n",
-    "- `poll_interval`* - Interval (in seconds) to poll the Kafka input topic for data\n",
-    "- `max_batch_size`* - Max batch size of data (max number of logs) to ingest into streamz with each `poll_interval`\n",
-    "- `data` - The input dataset to use for this streamz example. This is a filepath to text file containing lines of text to be processed for inference (i.e. log file)\n",
-    "\n",
-    "``*`` = More information on these parameters can be found in the streamz [documentation](https://streamz.readthedocs.io/en/latest/api.html#streamz.from_kafka_batched).\n",
+    "The Dockerfile contains an ENTRYPOINT which calls [entrypoint.sh](https://github.com/rapidsai/clx/blob/branch-0.17/examples/streamz/scripts/entrypoint.sh) to:\n",
+    "1. Configure and install Kafka\n",
+    "2. Run Kafka broker on `localhost:9092` and Zookeeper on `localhost:2181`\n",
+    "3. Start Dask Scheduler on `localhost:8786`\n",
+    "4. Start Dask CUDA Worker (one worker per GPU)\n",
+    "\n",
+    "**Steps to run workflow**\n",
+    "\n",
+    "1. Access docker container\n",
+    "\n",
+    "   `\n",
+    "   docker exec -it clx_streamz bash\n",
+    "   `\n",
+    "   \n",
+    "2. Create kafka topics for the clx_streamz workflows that we want to run and publish input data. \n",
+    "\n",
+    "    `\n",
+    "    bash $CLX_STREAMZ_HOME/scripts/kafka_topic_setup.sh --help\n",
+    "    `\n",
+    "    ```\n",
+    "    Usage: kafka_topic_setup.sh [POS]... [ARG]...\n",
+    "\n",
+    "    Example-1: bash kafka_topic_setup.sh -b localhost:9092 -i cybert_input -o cybert_output -d /opt/clx_streamz/data/cybert_input.csv\n",
+    "    Example-2: bash kafka_topic_setup.sh -i cybert_input -o cybert_output -d /opt/clx_streamz/data/cybert_input.csv\n",
+    "    Example-2: bash kafka_topic_setup.sh -i cybert_input -o cybert_ouput\n",
+    "    \n",
+    "    This script configures the kafka topic, such as creating and loading data or just the topic creation.\n",
+    "    \n",
+    "    Positional:\n",
+    "      -b,  --broker          Kafka broker. Default value is localhost:9092\n",
+    "      -i,  --input_topic     Input kafka topic\n",
+    "      -o,  --output_topic    Output kafka topic\n",
+    "      -d,  --data_path       Input data filepath\n",
+    "    \n",
+    "      -h, --help             Print this help\n",
+    "    ```\n",
+    "    \n",
+    "    ```\n",
+    "    \n",
+    "    bash $CLX_STREAMZ_HOME/scripts/kafka_topic_setup.sh \\\n",
```

Comment:
Is it okay to run this using docker exec `docker exec clx-streamz bash -c <command>` without having to log into the container?

Reply:
sure, we can do it. @efajardo-nv any thoughts?

Reply:
done